code/__DEFINES/wiremod.dm

```diff
@@ -114,6 +116,11 @@
 #define COMP_PROC_GLOBAL "Global"
 #define COMP_PROC_OBJECT "Object"
 
+// Bar overlay component
+#define BAR_OVERLAY_LIMIT 10
```

Comment:
Why did you move these out? They're only needed in that one file.

Reply:
since I do it for other components, e.g. arrest consoles, so I thought it made sense to move the define out to the wiremod.dm file.